myhdl/_Cosimulation.py

```diff
@@ -137,7 +137,10 @@ def _get(self):
                     if next >= (1 << (s._nrbits-1)):
                         next |= (-1 << s._nrbits)
             except ValueError:
-                next = intbv(0)
+                if v in 'xX':
+                    next = s._init
```

Comment:
I might have missed this in a conversation, what is the thinking behind this change?